SIL.Media/FFmpegRunner.cs

```diff
@@ -96,7 +97,23 @@ private static string LocateFFmpeg()
 				if (File.Exists(exePath))
 					return exePath;
 			}
-			return null;
+
+			//Locate may be called multiple times, we don't want to run this command every time.
+			if (_ffmpegOnPath == null)
```

Comment:
@tombogle and @andrew-polk would you review this for me?

Reply:
LGTM
<!-- Reviewable comment -OC-K4eQ7ko6GOF5vTZV:bb6ywmx -->
<!-- Sent from Reviewable.io -->

Reply:
yeah I  think if someone wants that they can add it, but it would be a bit of a pain to add as it needs to be backward compatible

Reply:
Wasn't too painful in a backwards compatible way and seemed better to add now rather than to need it later and have to add it retroactively. 
<!-- Reviewable comment -OC4p9c66F2nGQfS_r7f:b-a1i9z7 -->
<!-- Sent from Reviewable.io -->